src/app/ubs/ubs/components/ubs-order-form/ubs-order-form.component.ts

```diff
@@ -41,7 +42,8 @@ export class UBSOrderFormComponent implements OnInit, AfterViewInit, DoCheck, On
     private cdr: ChangeDetectorRef,
     private shareFormService: UBSOrderFormService,
     private localStorageService: LocalStorageService,
-    private store: Store
+    private store: Store,
```

Comment:
[Member 'store: Store' is never reassigned; mark it as `readonly`.](https://sonarcloud.io/project/issues?pullRequest=3407&open=AZKVdKLjhY9BiVF5AhU9&id=ita-social-projects-green-city-client)